README.md

```diff
@@ -10,8 +10,9 @@
   - iOS, 
   - NodeJS
   - Python (looking for a maintainer)  
+- Overview of [implemented Aries protocols](./aries.md)
 - Aries Framework Rust requires [mediator agency](https://github.com/hyperledger/aries-rfcs/blob/master/concepts/0046-mediators-and-relays/README.md).
-  Currently, the only such agency is [NodeVCX Agency](https://github.com/AbsaOSS/vcxagencynode/).  
+  Currently, the only such agency is [NodeVCX Agency](https://github.com/AbsaOSS/vcxagencynode/).    
```

Comment:
BTW, this statement seems too strong. I can easily imagine some developing another agency and being offended by this :)